tests/pytests/test_viking_drivers.py

```diff
@@ -0,0 +1,109 @@
+                     'TKFRAME_-27002_RELATIVE': 'VO1_PLATFORM',
+                     'INS-27002_CK_FRAME_ID': -27000.0,
+                     'TKFRAME_-27002_AXES': [1., 2., 3.],
+                     'TKFRAME_-27002_SPEC': 'ANGLES',
+                     'FRAME_-27002_CLASS_ID': -27002.0,
+                     'FRAME_-27002_CLASS': 4.0,
+                     'TKFRAME_-27002_UNITS': 'DEGREES',
+                     'BODY499_POLE_DEC': [52.8865, -0.0609,  0.    ],
+                     'BODY499_POLE_RA': [ 3.1768143e+02, -1.0610000e-01,  0.0000000e+00],
+                     'BODY499_PM': [176.63      , 350.89198226,   0.        ]
+                     },
+    'InstrumentPointing': {'TimeDependentFrames': [-27000, 2, 1],
+                           'CkTableStartTime': -679343589.99066,
+                           'CkTableEndTime': -679343589.99066,
+                           'CkTableOriginalSize': 1,
+                           'ConstantFrames': [-27002, -27000],
+                           'ConstantRotation': [-3.97934996888762e-04, 0.99992928417985, -0.011885633652183, -0.99999976485974,
+                                                -4.04545016850005e-04, -5.53736215647129e-04,-5.58505331602587e-04, 0.011885410506373,
+                                                0.99992921003884],
+                           'EphemerisTimes': [-679343589.99066],
+                           'Quaternions': [[0.027631069885584,-0.042844653015238,0.63550118209549,0.77041489292473]],
+                           'AngularVelocity': [[0.0,0.0,0.0]]},
+    'BodyRotation': {'TimeDependentFrames': [10014, 1],
+                     'CkTableStartTime': -679343589.99066,
+                     'CkTableEndTime': -679343589.99066,
+                     'CkTableOriginalSize': 1,
+                     'EphemerisTimes': [-679343589.99066],
+                     'Quaternions': [[-0.7279379541667,0.013628253415821,0.31784329352737,-0.60736829547822]],
+                     'AngularVelocity': [[3.16266124745133e-05,-2.87736941823106e-05,5.6534196515818e-05]]},
+    'InstrumentPosition': {'SpkTableStartTime': -679343589.99066,
+                           'SpkTableEndTime': -679343589.99066,
+                           'SpkTableOriginalSize': 1,
+                           'EphemerisTimes': [-679343589.99066],
+                           'Positions': [[3804.7657059701,-29029.379045591,-8152.7028148086]],
+                           'Velocities': [[0.61887065223952,-0.54866918471373,-0.037514399215278]]},
+   'SunPosition': {'SpkTableStartTime': -679343589.99066,
+                   'SpkTableEndTime': -679343589.99066,
+                   'SpkTableOriginalSize': 1,
+                   'EphemerisTimes': [-679343589.99066],
+                   'Positions': [[242154815.26715,36901072.557543,10366793.725074]],
+                   'Velocities': [[-4.6653907628395,19.836822435663,9.2246820568878]]}}
+
+
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
Do not autouse this. There can be weird things that happen with autouse fixtures and unittest testCases.